libethereum/Executive.h

```diff
@@ -78,6 +78,21 @@ class StandardTrace
 	DebugOptions m_options;
 };
 
+
+struct AccountSnapshot
+{
+	bool exists = false;
+	bool isCreation = false;
+	int nonceInc = 0;
+	Address address;
```

Comment:
Also it seems somewhat confusing that address here serves two purposes - could be value receiver, could be created address. I suggest at least a comment here about that

The same for caller member